src/main/java/me/coley/recaf/ui/controls/popup/DragPopup.java

```diff
@@ -15,6 +15,9 @@
  * @author Matt
  */
 public class DragPopup {
+	private static double lastX = -1D;
+	private static double lastY = -1D;
```

Comment:
Is there a reason why we would want to retain the position and show new popups at the same location?

If a popup is originating from a `Node` that does not match this position _(Or if the user moves the window on the screen)_ then wouldn't that be confusing? I'm not sure I understand why this would be beneficial.

Reply:
I was thinking that by making popups store their location it could make the experience a bit better when for example continuously getting decompiler errors on your way because of some well obfuscated code while only trying to look at something specific, but now that you brought up the other side of things I do agree it could get confusing and afterall there surely is a better solution to something like this.